client/reader/controller.js

```diff
@@ -19,7 +19,8 @@ var i18n = require( 'lib/mixins/i18n' ),
 	i18n = require( 'lib/mixins/i18n' ),
 	TitleStore = require( 'lib/screen-title/store' ),
 	titleActions = require( 'lib/screen-title/actions' ),
-	FeedSubscriptionActions = require( 'lib/reader-feed-subscriptions/actions' );
+	FeedSubscriptionActions = require( 'lib/reader-feed-subscriptions/actions' ),
+	urlHelper = require( 'reader/url-helper' );
```

Comment:
Would it make sense to include this resource in `lib/route/path`? There are some similar helpers there. Or even a new `lib/route/reader.js`.